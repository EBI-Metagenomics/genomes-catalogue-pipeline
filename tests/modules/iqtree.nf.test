nextflow_process {

    name "Test Process IQTREE"
    script "modules/iqtree.nf"
    process "IQTREE"

    test("Should run without failures for bac120") {

        when {
            params {
                outdir = "tests/results"

            }
            process {
                """
                input[0] = file("$baseDir/tests/modules/fixtures/gtdbtk/gtdbtk.bac120.msa.fasta.gz")
                input[1] = "bac120"
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
