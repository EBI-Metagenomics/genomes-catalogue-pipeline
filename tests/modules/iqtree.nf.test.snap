{
    "Should run without failures for bac120": {
        "content": [
            {
                "tree": [
                    [
                        "bac120",
                        "bac120_iqtree.nwk:md5,91f5c0b85baf19933a2575b79d897df0"
                    ]
                ]
            }
        ],
        "timestamp": "2023-02-13T13:41:19+0000"
    }
}